user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleDetailsComponent.java

```diff
@@ -193,10 +193,8 @@ public void setContext(Context context) {
     if (context.projectId().isEmpty()) {
       throw new ApplicationException("no project id in context " + context);
     }
-
     this.context = context;
-    ExperimentId experimentId = context.experimentId()
-        .orElseThrow(() -> new ApplicationException("no experiment id in context " + context));
+    ExperimentId experimentId = context.experimentId().get();
```

Comment:
what is the idea behind this change?

Reply:
Since we already throw an exception in the beginning of the setContext method if no experimentId was provided it's unnecessary to check and do so again here